src/Link.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+final class Link
```

Comment:
`Link` will be use for breadcrumbs only?

Reply:
I don't know yet, i have to develop the library, i would prefer to leave it like that, if it is used in another component great, otherwise in the end i will rename it.

Reply:
Can we use class of `A` tag instead of this object?

Reply:
Nope.